src/docquery/cmd/scan.py

```diff
@@ -22,6 +22,10 @@ def build_parser(subparsers, parent_parser):
 
     parser.add_argument("path", type=str, help="The file or directory to scan")
 
+    parser.add_argument(
+        "--ocr", choices=["tesseract", "easyocr", None], default=None, help="The orc engine you would like to use"
```

Comment:
Is there a way for a user to specify "do not use OCR" (i.e. always hit the Dummy case)?